bin/scrape-pokeapi.ts

```diff
@@ -140,6 +142,11 @@ export async function scrapePokeapi(): Promise<void> {
         key: (item) => item.stat.name,
         value: (item) => item.base_stat,
       });
+      const efforts = toObject({
+        data: detail.stats,
+        key: (item) => item.stat.name,
+        value: (item) => item.effort,
+      });
```

Comment:
Let's change these names like `special-attack` ➡️  `spAttack` to match the other keys